src/main/cli/cli.c

```diff
@@ -263,7 +263,8 @@ static const char * const featureNames[] = {
     "RANGEFINDER", "TELEMETRY", "", "3D", "RX_PARALLEL_PWM",
     "RX_MSP", "RSSI_ADC", "LED_STRIP", "DISPLAY", "OSD",
     "", "CHANNEL_FORWARDING", "TRANSPONDER", "AIRMODE",
-    "", "", "RX_SPI", "", "ESC_SENSOR", "ANTI_GRAVITY", "DYNAMIC_FILTER", NULL
+    "", "", "RX_SPI", "", "ESC_SENSOR", "ANTI_GRAVITY", "DYNAMIC_FILTER",
+    "RX_MSP_OVERRIDE", NULL
```

Comment:
Not sure that making this a feature is the right thing to do - features are relatively hard to work with and 'legacy' - we can probably just use `msp_override_passthrough_channels != 0` to check if this is enabled or not.

Reply:
Sounds good